core/src/main/scala/org/scalafmt/config/Newlines.scala

```diff
@@ -36,6 +36,24 @@ import metaconfig.ConfigReader
   *     n =>
   *       consume(n)
   *   }
+  * @param afterCurlyLambda
+  *   If `never` (default), it will remove any extra lines below curly lambdas
+  *   {{{
+  *   something.map { x =>
+  *
+  *     f(x)
+  *   }
+  *   }}}
+  *   will become
+  *   {{{
+  *   something.map { x =>
+  *     f(x)
+  *   }
+  *   }}}
+  *
+  *   If `always`, it will always add one empty line (opposite of `never`).
```

Comment:
Now that I think about it, I'm not sure `always` is the correct name for this. It isn't really "always" since the single-line split takes precedence. `force` might work, but it also isn't really "forced".